src/Config.ts

```diff
@@ -315,7 +310,7 @@ export class Config implements ConfigParams, ParserConfig {
   }
 
   private static warnDeprecatedOptions(options: Partial<ConfigParams>) {
-    Config.warnDeprecatedIfUsed(options.binarySearchThreshold, 'binarySearchThreshold', '1.1')
+    // Config.warnDeprecatedIfUsed(options.binarySearchThreshold, 'binarySearchThreshold', '1.1')
```

Comment:
I think it's a good idea to keep it commented as an example of how to use deprecation warnings. However, I'd add an explanation for reviewers that it's not a leftover.
```suggestion
    // an example of deprecation warning
    // Config.warnDeprecatedIfUsed(options.binarySearchThreshold, 'binarySearchThreshold', '1.1')
```

Reply:
Good idea